brushfire-finatra/src/main/scala/com/stripe/brushfire/finatra/BrushfireServer.scala

```diff
@@ -7,7 +7,7 @@ import com.twitter.finagle.http._
 import com.twitter.bijection._
 
 class BrushfireServer extends FinatraServer {
-  def score[K, V, T: Semigroup, P](root: String, trees: Iterable[Tree[K, V, T]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
+  def score[K, V, T: Semigroup, P](root: String, trees: Iterable[Tree[K, V, T, Unit]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
```

Comment:
We use annotations in a model server to guide traversals (eg for how you deal with missing features, either organically missing or deleted as part of some feature importance scheme). In general I'm pretty sure that people will find creative uses for them.